package.json

```diff
@@ -68,12 +68,11 @@
     "focus-visible": "^5.2.0",
     "glob": "^7.1.6",
     "lodash.clonedeep": "^4.5.0",
-    "lodash.debounce": "^4.0.8",
+    "throttle-debounce": "^4.0.8",
```

Comment:
Could you please update the version of `throttle-debounce` to its latest  version: "^3.0.1".
You will then need to update the lock file by running `pnpm install`, and committing.